boot/bootutil/src/encrypted.c

```diff
@@ -527,16 +527,17 @@ boot_enc_decrypt(const uint8_t *buf, uint8_t *enckey)
     }
 
     /* is EC point uncompressed? */
-    if (buf[0] != 0x04) {
-        return -1;
-    }
+//    if (buf[0] != 0x04) {
+//        return -1;
+//    }
```

Comment:
I might have mentioned it before but commented out code is not acceptable, fix the ocurrences and I will review further.

Reply:
cleaned